fakebook/src/app/service/auth.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { AuthService } from './auth.service';
+
+describe('AuthService', () => {
+  let service: AuthService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(AuthService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
```

Comment:
we don't add unit tests that pass, only unit tests that fail!
make the constructor throw an error, and don't add isAuthenticated since it's not relevant to any tests

Reply:
Constructor throws error in new PR.